packages/forms-web-app/src/pages/projects/register/index/controller.test.js

```diff
@@ -114,5 +114,23 @@ describe('projects/register/index/controller', () => {
 			expect(res.status).toHaveBeenCalledWith(404);
 			expect(responseWithStatus.render).toHaveBeenCalledWith('error/not-found');
 		});
+		it('should redirect to not found route if registration period has closed and openRegistrationCaseReferences is empty', async () => {
+			getAppData.mockImplementation(() =>
+				Promise.resolve({
+					resp_code: 404,
+					data: {
+						DateOfRepresentationPeriodOpen: '2022-04-01',
+						DateOfRelevantRepresentationClose: '2022-07-01',
+						openRegistrationCaseReferences: ''
+					}
```

Comment:
It doesn't _really_ matter for the test itself, but if the `resp_code` you mocked here was 404, you wouldn't get any data back ;) 
So I would probably change that to `200` so it makes sense in the context of the test